### PR TITLE
CONN-1203:Intergarte AdminGUI Certificates page with CONNECTDirectConfig

### PR DESCRIPTION
**Pull request by alameluchidambaram**

### CODE REVIEW

**Comment by vsaibabu** [comment]
retest

**Comment by christophermay07** [comment]
retest this please

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/proxy/DirectConfigProxyWebServiceUnsecuredImpl.java [hunk 1]
```diff
@@ -25,59 +25,111 @@
...
+@SuppressWarnings("unchecked")
 @Service
 public class DirectConfigProxyWebServiceUnsecuredImpl implements DirectConfigProxy {
 
     private final WebServiceProxyHelper oProxyHelper = new WebServiceProxyHelper();
-    
-    private final Class directConfigClazz = ConfigurationService.class;
-    
+
+    List<Certificate> certList = new ArrayList<Certificate>();
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/proxy/DirectConfigProxyWebServiceUnsecuredImpl.java:53]: What is this used for?

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/direct/DirectServiceImpl.java [hunk 2]
```diff
@@ -85,57 +90,94 @@ public void deleteDomain(Domain domain) {
...
 
     @Override
     public void deleteAgent(DirectAgent agent) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+        throw new UnsupportedOperationException("Not supported yet."); // To change body of generated methods, choose
+                                                                       // Tools | Templates.
     }
 
+    /*
+     * @Override public List<Certificate> getCertificatesForOwner(GetCertificatesForOwner certForOwner) {
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/direct/DirectServiceImpl.java:116]: remove this if we're not implementing it.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/proxy/DirectConfigProxyWebServiceUnsecuredImpl.java (hunk 1 above)
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/proxy/DirectConfigProxyWebServiceUnsecuredImpl.java:53]: Removed in PR #849

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/direct/DirectServiceImpl.java (hunk 2 above)
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/direct/DirectServiceImpl.java:116]: Removed in PR #849

**Comment by christophermay07** [comment]
Closing without merge; this code will be merged into CONNECT as part of PR #849

**Pull request closed without merging**